packages/page-statistics/src/containers/DonutChartContainer/DonutChartContainer.tsx

```diff
@@ -1,34 +1,44 @@
 import { useDonutColor } from '@webb-dapp/page-statistics/hooks';
+import { ProposalStatus } from '@webb-dapp/page-statistics/provider/hooks';
 import { Card, DropdownMenu, Label, TitleWithInfo } from '@webb-dapp/webb-ui-components/components';
 import { Typography } from '@webb-dapp/webb-ui-components/typography';
-import { FC, useMemo, useState } from 'react';
+import { FC, useMemo } from 'react';
 import { ChartProps, Doughnut } from 'react-chartjs-2';
 
 import { DonutDataType } from './types';
 
-const labels = [DonutDataType.Open, DonutDataType.Rejected, DonutDataType.Signed, DonutDataType.Accepted];
 const data = [9, 2, 3, 5];
+export type TimeRange = 'Day' | 'Week' | 'Year' | 'All Time';
 
-export const DonutChartContainer = () => {
-  const menuOptions = useMemo(() => ['Day', 'Week', 'Year', 'All Time'], []);
-  const [selectIndex, setSelectIndex] = useState(0);
+type ProposalsOverviewProps = {
+  statsMap: Record<ProposalStatus, number>;
+  isLoading: boolean;
+  timeRange: TimeRange;
+  setTimeRange: (timeRange: TimeRange) => void;
+};
+export const DonutChartContainer: FC<ProposalsOverviewProps> = ({ isLoading, setTimeRange, statsMap, timeRange }) => {
+  const menuOptions = useMemo<TimeRange[]>(() => ['Day', 'Week', 'Year', 'All Time'], []);
+  const selectIndex = useMemo(() => menuOptions.indexOf(timeRange), [menuOptions, timeRange]);
```

Comment:
Prefer to not have these as `useMemo` since these values are hardcoded. Can we just have them be equal to the value?

```suggestion
  const menuOptions = ['Day', 'Week', 'Year', 'All Time'];
  const selectIndex = () => menuOptions.indexOf(timeRange);
```

Reply:
The memo hook ensures the render times of the component. I agreed with you that these are hardcoded values, we can move the declaration outside of the component scope 👍.